contractcourt/chain_watcher.go

```diff
@@ -124,7 +125,9 @@ type chainWatcher struct {
 func newChainWatcher(chanState *channeldb.OpenChannel,
 	notifier chainntnfs.ChainNotifier, pCache WitnessBeacon,
 	signer lnwallet.Signer, isOurAddr func(btcutil.Address) bool,
-	markChanClosed func() error) (*chainWatcher, error) {
+	markChanClosed func() error,
```

Comment:
This constructor is getting a little cray, not a blocker tho. could maybe refactor using a watcher config in the future

Reply:
Added a config